client/src/App.js

```diff
@@ -3,12 +3,13 @@ import logo from './logo.svg';
 import './App.css';
 import BFSNav from './components/Navbar';
 import PaymentPortal from './pages/ParentPortal'
+import PaymentPage from './pages/PaymentPage'
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
+        {/* <header className="App-header">
```

Comment:
Let's revert this change and worry about replacing App.js later.

`git checkout -- client/src/App.js` to revert!

Reply:
I ran that command but nothing seems to have changed?

Reply:
I think you might need to run

<code> git checkout master -- client/src/App.js </code>